org.opent2t.sample.multisensor.superpopular/com.wink.sensorpod/js/thingTranslator.js

```diff
@@ -70,10 +70,15 @@ class StateReader {
 }
 
 function convertDeviceDateToTranslatorDate(unixTimestamp) {
-    // Date takes a number of milliseconds, so convert seconds to milliseconds
-    var datetime = new Date(unixTimestamp * 1000);
+    // Only convert if input is valid numeric value
+    if (unixTimestamp && !isNaN(unixTimestamp)) { 
+        // Date takes a number of milliseconds, so convert seconds to milliseconds
+        var datetime = new Date(unixTimestamp * 1000);
+        return datetime.toISOString();
+    }
 
-    return datetime.toISOString();
+    // return all other invalid input as-is since we can't convert to DateTime
+    return unixTimestamp;
```

Comment:
>// return all other invalid input as-is since we can't convert to DateTime 

The lastchanged resource is defined as an opent2t.r.timestamp, which specifies that the string should be in the ISO8601 format.

`"description": "Using ISO 8601 datetime format (e.g: 2007-04-05T14:30Z, 2007-04-05T14:30+09:00)"`

It would be good to understand why the _changed_at property isn't always showing up in the Wink payload. If it happens for devices that haven't reported a value yet, for example, it might be a good idea to fall back to the _updated_at property instead of just returning 'undefined' here.

See the Wink documentation for the difference between the two:

http://docs.winkapiv2.apiary.io/reference/device/desired-state-and-last-reading